src/ProjectMSDLApplication.cpp

```diff
@@ -36,15 +36,21 @@ void ProjectMSDLApplication::initialize(Poco::Util::Application& self)
 
     loadConfiguration(PRIO_DEFAULT);
 
+    // create config dir
+    Poco::Path configDir(config().getString("application.configDir"));
+    Poco::File(configDir).createDirectory();
+    SDL_LogInfo(SDL_LOG_CATEGORY_APPLICATION, "Config dir '%s'\n", configDir.toString().c_str());
+
+    // set default config
+    config().setString("projectM.presetExcludeFile", Poco::Path(configDir, "exclude.txt").toString());
```

Comment:
I didn't see anything else calling `set`, but I did this so both the wrapper and and render loop can get the config with the right path.